t/lib/MemcachedTest.pm

```diff
@@ -72,6 +73,16 @@ sub mem_stats {
     return $stats;
 }
 
+sub mem_move_time {
```

Comment:
think I see how the wording on my issue may have complicated things slighty. I want to be able to pause and resume time, /as well as/ move it. So the act of moving time here should simply b `debugtime $mover\r\n`, there shouldn't need to be pause/resume.

I'll add more notes in the top level review bit.